docs/api.md

```diff
@@ -4039,6 +4042,7 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Project (object)
 + id: `1` (number) - The numerical ID of the Project.
 + name: `Default Project` (string, required) - The name of the Project.
++ description: `Description of this Project to show on Central.` (string, optional) - The description of the Project.
```

Comment:
Just a thought I had, it might be useful to say explicitly that this will be rendered as Markdown in Frontend?